src/main/java/me/dinowernli/grpc/prometheus/ServerMetrics.java

```diff
@@ -160,15 +176,27 @@ public void recordLatency(double latencySec, Metadata metadata) {
               .register(registry);
 
       if (configuration.isIncludeLatencyHistograms()) {
+
+        final List<String> labels = new ArrayList<String>();
+        labels.addAll(defaultRequestLabels);
+        labels.addAll(configuration.getSanitizedLabelHeaders());
+        if (configuration.addCodeLabelToHistograms()) {
+          labels.add(statusCodeLabel);
+          this.addCodeLabelToHistograms = true;
```

Comment:
Would it be simpler to just set `this.addCodeLabelToHistograms = configuration.addCodeLabelToHistograms()` ?

Reply:
Done)